src/main/helpers.ts

```diff
@@ -0,0 +1,116 @@
+import { dungeonsByMapId, instanceDifficulty, InstanceDifficultyType, instanceEncountersById, instanceNamesByZoneId, raidInstances, VideoCategory, zones } from "./constants";
+import { Metadata } from "./logutils";
+import { RaidInstanceType } from "./types";
+
+/**
+ * Get a result text appropriate for the video category that signifies a
+ * win or a loss, of some sort.
+ */
+export const getVideoResultText = (category: VideoCategory, isGoodResult: boolean): string => {
+
+    // Not sure how we can decide who won or lost yet.
```

Comment:
This probably needs an update. "Non-trivial to determine who won a BG/SoloShuffle so just don't report it" -- something like that